admin/class-yoast-notification.php

```diff
@@ -52,6 +60,165 @@ public function get_id() {
 	}
 
 	/**
+	 * Retrieve nonce identifier.
+	 *
+	 * @return null|string Nonce for this Notification.
+	 */
+	public function get_nonce() {
+		if ( $this->options['id'] && empty( $this->options['nonce'] ) ) {
+			$this->options['nonce'] = wp_create_nonce( $this->options['id'] );
+		}
+
+		return $this->options['nonce'];
+	}
+
+	/**
+	 * Get the type of the notification
+	 *
+	 * @return string
+	 */
+	public function get_type() {
+		return $this->options['type'];
+	}
+
+	/**
+	 * Priority of the notification
+	 *
+	 * Relative to the type
+	 *
+	 * @return float 0-1
```

Comment:
This 0-1 is very lonely and confusing for people reading the code. Maybe `Returns a priority between 0 and 1`.